Mobile/ShopLens/Droid/SpeechActivity.cs

```diff
@@ -0,0 +1,171 @@
+﻿
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Widget;
+using Android.Speech;
+using Android.Speech.Tts;
+using Java.Util;
+using Android.Runtime;
+
+namespace ShopLens.Droid
+{
+    [Activity(Label = "SpeechActivity")]
+    public class SpeechActivity : Activity, TextToSpeech.IOnInitListener
+    {
+        bool isRecording;
+        readonly int VOICE = 10;
```

Comment:
what exactly is this? Voice "request code"? The name could be a little more descriptive I guess